tests/PySys/tedge_mapper_c8y/tedge_mapper_c8y_child_device_measurement/run.py

```diff
@@ -0,0 +1,47 @@
+from environment_c8y import EnvironmentC8y
+
+import time
+import json
+import os
+
+
+class TedgeMapperC8yChildDevice(EnvironmentC8y):
+    def setup(self):
+        super().setup()
+        child_device_fragment = self.cumulocity.get_child_device_of_thin_edge_device_by_name(
+            self.project.device, "thin-edge-child")
+        self.assertThat('actual == expected',
+                        actual=child_device_fragment, expected=None)
+        self.addCleanupFunction(self.test_cleanup)
+
+    def execute(self):
+        self.startProcess(
+            command=self.tedge,
+            arguments=["mqtt", "pub",
+                       "tedge/measurements/thin-edge-child",
+                       '{"temperature": 12, "time": "2021-06-15T17:01:15.806+02:00"}'],
+            stdouterr="tedge_pub",
+            environs=os.environ
```

Comment:
Do we need something from the environment here?

Reply:
Yes. `tedge mqtt` utility expects the `USER_HOME` env without which it would fail. In all other tests we've gotten around that issue by calling `tedge mqtt` with `sudo`. Did this just to avoid that sudo.